src/operational-security/human-centered-security/travel-security.md

```diff
@@ -0,0 +1,94 @@
+2. Consider the sensitivity of work to be conducted during travel and adjust security measures accordingly
+3. Consult reliable sources such as government travel advisories and internal threat intelligence
+
+### Device Preparation
+
+1. Use dedicated travel devices whenever possible, with minimal data and applications
+2. Update all software and security patches before departure
+3. Enable full-disk encryption on all devices
+4. Install and test VPN software prior to departure
+5. Consider using temporary travel accounts with limited access to organizational resources
+6. Backup all data before departure and store the backup securely
+
+### Documentation and Emergency Planning
+
+1. Create copies of important documents (passport, visas, insurance)
+2. Establish emergency communication protocols and points of contact
+3. Share itinerary details with appropriate team members
+4. Register with embassy or consular services when traveling to high-risk locations
+
+## In-Transit Security
+
+1. Maintain physical control of devices and sensitive items at all times
+2. Be aware of shoulder surfing when working on sensitive information
+3. Avoid connecting to public or hotel Wi-Fi networks without VPN protection
+4. Disable Bluetooth, Wi-Fi, and other wireless connectivity when not in use
+5. Consider using privacy screens to prevent visual data leakage
+
+## On-Location Security
+
+### Physical Security
+
+1. Store sensitive devices in hotel safes when not in use, or maintain physical possession
+2. Be aware of surroundings and potential surveillance or targeting
+3. Avoid discussing sensitive business matters in public spaces
+4. Use privacy screens when working in public areas
+
+### Digital Security
+
+1. Connect to corporate networks only through secure VPN channels
+2. Be cautious of USB charging stations which may present data theft risks
+3. Report any suspicious activity or security incidents immediately
+4. Exercise extra caution when connecting to Wi-Fi networks
+5. Use mobile hotspots from trusted providers when possible
+
+## Post-Travel Procedures
+
+1. Inspect devices for signs of tampering
+2. Change passwords used during travel
+3. Run security scans on all devices before reconnecting to corporate networks
+4. Report any suspicious incidents or security concerns
+5. Debrief with security team about any potential security issues encountered
+
+## Special Considerations for High-Risk Locations
+
+1. Consider using "burner" devices for travel to high-risk locations
+2. Establish more frequent check-in protocols
+3. Be aware of local surveillance capabilities and legal requirements for device inspection
+4. Prepare for potential device confiscation at borders by minimizing sensitive data
+5. Consider using end-to-end encrypted communication methods
+
+## Web3-Specific Travel Security
+
+1. Use hardware wallets for storing crypto assets and keep them physically secure
```

Comment:
Recommend not travelling with hardware wallets if they are not needed throughout duration of travel